src/components/BlogForm/TitleEntry.tsx

```diff
@@ -81,23 +76,22 @@ const TitleEntry = () => {
                 fieldState: { error },
               }: any) => (
                 <TextField
-                  label="New Title..."
+                  label="New Title"
                   fullWidth
                   value={value}
                   onChange={onChange}
                   error={!!error}
                   helperText={!!error && "Title is required"}
-                ></TextField>
+                />
               )}
             />
             <Button
               type="submit"
               sx={{
                 mt: 2,
-                background: theme.palette.primary.main,
-                color: "white",
+                color: theme.palette.primary.main,
+                // color: "white",
```

Comment:
why are you defining the color? it should be set by default to the correct styles. also delete the comment

Reply:
dude, idk what to tell you with put defining its a white button with green text